notebooks/en/finetune_t5_for_search_tag_generation.ipynb

```diff
@@ -0,0 +1,7540 @@
+{
```

Comment:
<p>few more would be appreciated, actually it would be very meta if you also added an inference example of this repository and some popular HF repositories as well!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>